src/style-util.js

```diff
@@ -325,21 +325,21 @@ const CSS_BUILD_ATTR = 'css-build';
  * @return {string} Can be "", "shady", or "shadow"
  */
 export function getCssBuild(element) {
-  if (element['__cssBuild'] === undefined) {
```

Comment:
This was a recent change to fix an undefined bug. Are we sure Closure now properly understands this?

Reply:
Yes, this PR also adds an extern for __cssBuild. Both the extern or the [] style access will cause Closure not to rename the property. I switched this to the externs style for consistency with all the other properties.

Reply:
Ah I see. Thanks for the explanation!